### PR TITLE
fix: (Platform) add tooltip to the components

### PR DESCRIPTION
**Pull request by Lokanathan-k**
platform tool tip adding

#### Please provide a link to the associated issue.
fixes https://github.com/SAP/fundamental-ngx/issues/3697

#### Please provide a brief summary of this pull request.

Actual: None of the info label examples have a tooltip.

Expected: info label should have a tooltip on hover.

#### Please check whether the PR fulfills the following requirements

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/master/CONTRIBUTING.md
- [x] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist

Documentation checklist:
- [x] Documentation Examples
- [x] Stackblitz works for all examples

### CODE REVIEW

**Review by Seamoo13 — APPROVED** [review]
LGTM

**Comment by InnaAtanasova** [comment]
@Lokanathan-k please use the appropriate way of writing the PR title

File: libs/platform/src/lib/components/info-label/info-label.component.html [hunk 1]
```diff
@@ -1,3 +1,10 @@
...
-	<ng-content></ng-content>
-</span>
\ No newline at end of file
+<fd-info-label
+    [type]="type"
+    [color]="color"
+    [glyph]="glyph"
+    [attr.aria-label]="ariaLabel"
+    [attr.aria-labelledby]="ariaLabelledBy"
+    [title]="title"
```
Comment (InnaAtanasova) [inline libs/platform/src/lib/components/info-label/info-label.component.html:7]: Why is in this case `title` and in the other cases `attr.title`?

File: apps/docs/src/app/platform/component-docs/platform-object-status/platform-object-status-example/platform-object-status-large-example.component.html [hunk 2]
```diff
@@ -55,6 +79,7 @@
     large="true"
     inverted="true"
     (click)="showAlert()"
+    label
```
Comment (InnaAtanasova) [inline apps/docs/src/app/platform/component-docs/platform-object-status/platform-object-status-example/platform-object-status-large-example.component.html:82]: empty?

File: apps/docs/src/app/platform/component-docs/platform-object-status/platform-object-status-example/platform-object-status-inverted-example.component.html [hunk 3]
```diff
@@ -1,24 +1,39 @@
...
 
 <br /><br />
 
-<fdp-object-status status="negative" inverted="true">Negative</fdp-object-status>
-<fdp-object-status status="critical" inverted="true">Critical</fdp-object-status>
-<fdp-object-status status="positive" inverted="true">Positive</fdp-object-status>
-<fdp-object-status status="informative" inverted="true">Informative</fdp-object-status>
-<fdp-object-status inverted="true">Default</fdp-object-status>
+<fdp-object-status status="negative" inverted="true" label="Negative"></fdp-object-status>
+<fdp-object-status status="critical" inverted="true" lable="Critical"></fdp-object-status>
```
Comment (InnaAtanasova) [inline apps/docs/src/app/platform/component-docs/platform-object-status/platform-object-status-example/platform-object-status-inverted-example.component.html:25]: Typo

File: apps/docs/src/app/platform/component-docs/platform-object-status/platform-object-status-example/platform-object-status-icon-text-example.component.html [hunk 4]
```diff
@@ -1,5 +1,5 @@
-<fdp-object-status status="negative" glyph="status-negative">Negative</fdp-object-status>
-<fdp-object-status status="critical" glyph="status-critical">Critical</fdp-object-status>
-<fdp-object-status status="positive" glyph="status-positive">Positive</fdp-object-status>
-<fdp-object-status status="informative" glyph="hint">Informative</fdp-object-status>
-<fdp-object-status glyph="to-be-reviewed">Default</fdp-object-status>
+<fdp-object-status status="negative" glyph="status-negative" lable="Negative" title="Negative"></fdp-object-status>
+<fdp-object-status status="critical" glyph="status-critical" lable="Critical" title="Critical"></fdp-object-status>
+<fdp-object-status status="positive" glyph="status-positive" lable="Positive" title="Positive"></fdp-object-status>
+<fdp-object-status status="informative" glyph="hint" lable="Informative" title="Informative"></fdp-object-status>
+<fdp-object-status glyph="to-be-reviewed" label="Default" title="Default"></fdp-object-status>
```
Comment (InnaAtanasova) [inline apps/docs/src/app/platform/component-docs/platform-object-status/platform-object-status-example/platform-object-status-icon-text-example.component.html:5]: Typo

File: apps/docs/src/app/platform/component-docs/platform-object-status/platform-object-status-example/platform-object-status-clickable-and-icon-example.component.html [hunk 5]
```diff
@@ -1,20 +1,44 @@
...
+    glyph="hint"
+    clickable="true"
+    (click)="showObjectStatus()"
+    lable="Informative"
+></fdp-object-status>
+<fdp-object-status
+    glyph="to-be-reviewed"
+    clickable="true"
+    (click)="showObjectStatus()"
+    lable="Default"
```
Comment (InnaAtanasova) [inline apps/docs/src/app/platform/component-docs/platform-object-status/platform-object-status-example/platform-object-status-clickable-and-icon-example.component.html:33]: Typo for `lable`. Should be `label`

**Review by InnaAtanasova — CHANGES_REQUESTED** [review]
Also the PR title should be following the standard we agreed on.

File: libs/platform/src/lib/components/info-label/info-label.component.html (hunk 1 above)
Comment (Lokanathan-k) [reply libs/platform/src/lib/components/info-label/info-label.component.html:7]: `attr.title` is already added in the core component in order not to duplicate i have added like as` title`

File: apps/docs/src/app/platform/component-docs/platform-info-label/platform-info-label-example/platform-info-label-aria-label-example.component.html [hunk 6]
```diff
@@ -1,19 +1,29 @@
...
+        <th id="name">Name</th>
+        <th>Grade</th>
+        <th id="status">Status</th>
+    </tr>
+    <tr>
+        <td>Alfreds Futterkiste</td>
+        <td>B</td>
+        <td>
+            <fdp-info-label
+                [type]="'numeric'"
```
Comment (sKudum) [inline apps/docs/src/app/platform/component-docs/platform-info-label/platform-info-label-example/platform-info-label-aria-label-example.component.html:21]: if possible shall we remove [] from binding

**Review by sKudum — APPROVED** [review]

**Review by InnaAtanasova — APPROVED** [review]
Screen Shot 2020-11-10 at 10 13 32 AM

Some leftover typos

**Comment by InnaAtanasova** [comment]
And please add proper naming for the PR titles.

**Pull request merged**